fastlane/Fastfile

```diff
@@ -1324,16 +1328,24 @@ end
 # Kicks off a Buildkite build
 # -----------------------------------------------------------------------------------
 def trigger_buildkite_release_build(branch:, beta:)
-  build_url = buildkite_trigger_build(
-    buildkite_organization: 'automattic',
-    buildkite_pipeline: 'woocommerce-ios',
-    branch: branch,
-    environment: { BUILDKITE_BETA_RELEASE: beta },
-    pipeline_file: 'release-builds.yml'
-  )
+    # If we're running on CI, we can directly start the release pipeline jobs within the same build
+    if is_ci
+      buildkite_pipeline_upload(
+        pipeline_file: BUILDKITE_RELEASE_PIPELINE_FILE_PATH,
+        env_file: BUILDKITE_SHARED_PIPELINE_VARS_FILE_PATH,
+        environment: { BUILDKITE_BETA_RELEASE: beta },
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/woocommerce/woocommerce-ios/blob/0f223b96ead709c307084ad27ac33338980a54b6/fastlane/Fastfile#L1336"></span>Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call.</td>
    </tr>
  </tbody>
</table>